docs/viplab3.0/computation_template.md

```diff
@@ -173,7 +179,7 @@ Note: `//` with text following until EOL is a comment,
               "metadata" : {
                 "guiType": "checkbox",
                 "name": "options",
-                "decription" : "This section describes the parameter",
+                "decription" : "Select output type"
```

Comment:
Gilt auch für dien anderen descriptions: Das ist das was einem beim angezeigt wird wenn man auf das i geht oder? Dann müssen auch die Werte beschrieben werden - es soll ja dem Nutzer helfen zu verstehen, was er tatsächlich auswählen kann. Evtl. ist DuMuX auch ein viel zu schweres Beispiel hier - da wissen wir ja gar nicht wie wir ein sinnvolles Beispiel bauen. Nimm doch eine Art "Hallo World" Container und füg dort Parameter hinzu.

Reply:
Ah, und falls du das Beispiel umbaust, es wäre gut den "content" in einem Kommentar zu decrypten (so wie beim c-beispiel oben), damit man es als Mensch nachvollziehen kann